packages/elements-core/src/components/Docs/HttpOperation/Request.tsx

```diff
@@ -129,7 +129,6 @@ const SecuritySchemes = ({ schemes }: { schemes: HttpSecurityScheme[][] }) => {
   if (!schemes.length) {
     return null;
   }
-  /* Note: not sure what to do about the nodeId below and how important it was to be tied to the scheme id like that... */
```

Comment:
🙄 🤣

Reply:
yeah I left a comment in the previous PR 😆